sass/abstracts/_structure.scss

```diff
@@ -3,6 +3,8 @@ $size__width--main: 940px;
 $size__width--post: 620px;
 
 $size__height--nav: 73px;
+$size__height--wave: 95px;
+$size__height--wave-huge: 7vw;
```

Comment:
Why is the height using vw units? Edit: I think it's something to do with the ratio of wider screen to taller image, is that right?